recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -906,6 +901,54 @@ def work(self):
         self.phase_image_emitter.emit(phase)
         self.meta_emitter.emit(meta)
 
+    def _check_exposure(self) -> None:
+        """
+        Check that all LF channels have the same exposure settings. If not, use the State0 exposure.
+
+        Parameters
+        ----------
+        settings:       (json) JSON dictionary conforming to MM SequenceSettings
+        """
+        logging.debug('Verifying exposure times...')
+        # parse exposure times
+        channel_exposures = []
+        for _, channel in enumerate(self.settings['channels']):
+            channel_exposures.append(channel['exposure'])
+
+        channel_exposures = np.array(channel_exposures)
+        if not np.all(channel_exposures == channel_exposures[0]):
+            # warn user that not all channels exposures were the same
+            logging.warning('The exposure times of each State are not equal!\nAcquiring with the exposure of State0...')
+
+            # setting all channel exposures to the exposure of State0
+            for i in range(len(self.settings['channels']) - 1):
+                self.settings['channels'][i + 1]['exposure'] = self.settings['channels'][0]['exposure']
```

Comment:
Consider changing to
```
for i in range(len(self.settings['channels'])):
    self.settings['channels'][i]['exposure'] = self.settings['channels'][0]['exposure']
```
so that a reader doesn't have to think through all of the off-by-one possibilities.

Reply:
That was the original code. My thought was that Python probably does not have compile-time-optiization that takes out `self = self`. But shouldn't matter anyway since this is cheap.

Reply:
Changed it back to the simpler way